tensilelite/Tensile/KernelWriterAssembly.py

```diff
@@ -8661,13 +8660,26 @@ def localWriteResetOffsets(self, kernel, internalPointerSwap, tP):
                                comment="Set LWA to first buffer offset"))
             self.vgprPool.checkIn(tmpvgpr)
         else:
-          numLwa = self.states.m.numVgprLocalWriteAddr
-          for i in range(numLwa):
-            module.add(VAndB32(
-                dst=vgpr("LocalWriteAddr%s+%u"%(tPM["tensorChar"], i)), \
-                src0=resetMask, \
-                src1=vgpr("LocalWriteAddr%s+%u"%(tPM["tensorChar"], i)), \
-                comment="reset to Red"))
+          module.add(VAndB32(
+            dst=vgpr("LocalWriteAddr%s+%u"%(tPM["tensorChar"], 0)), \
+            src0=resetMask, \
+            src1=vgpr("LocalWriteAddr%s+%u"%(tPM["tensorChar"], 0)), \
+            comment="reset to Red"))
+
+    numLwa = 0
+    if tP["isA"]:
+      numLwa = self.states.a.numVgprLocalWriteAddr
+    elif tP["isB"]:
+      numLwa = self.states.b.numVgprLocalWriteAddr
+    elif tP["isM"]:
+      numLwa = self.states.m.numVgprLocalWriteAddr
+
+    for i in range(1, numLwa):
+      module.add(VAddU32(
+        dst=vgpr("LocalWriteAddr%s+%u"%(tc, i)), \
```

Comment:
@TonyYHsieh, @alex391a is it okay to use `tc = tP["tensorChar"]` here? Previously this was `tPM["tensorChar"]`.